arches_lingo/src/arches_lingo/components/generic/ComponentManager/ComponentManager.vue

```diff
@@ -130,7 +136,8 @@ provide("refreshReportSection", refreshReportSection);
 
         <SplitterPanel
             v-if="editorState !== CLOSED"
-            :size="editorState === MINIMIZED ? 33 : 100"
+            :size="editorState === MINIMIZED ? 70 : 100"
```

Comment:
future nit: looks like we no longer need to calculate the size based on editorState. Locally a 50/50 split worked.